CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Snyk Security Changelog
 
+## [2.4.0]
+### Added
+Added a preview for generated Code Fixes
```

Comment:
@bastiandoetsch I wasnt sure how much in detail to go here. Is this summary enough?

Reply:
That's ok. I would maybe rephrase to "Improve UX of AI fixes by adding previews and options". But it's also ok as is. The version number though does not seem correct. I'd assume 2.6.0 would be the release version (currently, we're at 2.5.0).